drydock/templates/kustomized/tutor13/extensions/prometheus-stack/kustomization.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: monitoring
+
+helmCharts:
+- name: kube-prometheus-stack
+  repo: https://prometheus-community.github.io/helm-charts
+  version: 38.0.2
+  releaseName: prometheus
+  includeCRDs: true
+  valuesInline: {{ DRYDOCK_PROMETHEUS_VALUES }}
```

Comment:
Should we add this as optional?

Reply:
Does it break if `valuesInline` is empty? you can also do `{{ DRYDOCK_PROMETHEUS_VALUES | default({}) }}`.

One potential issue is that this will convert `DRYDOCK_PROMETHEUS_VALUES` to a json dict, if you have a lot of values then you end up with an extremely long line.

What thing that we can do is to assume `DRYDOCK_PROMETHEUS_VALUES` is a string:

``` config.yml
DRYDOCK_PROMETHEUS_VALUES: |
  defaultRules:
    create: true
    rules:
      alertmanager: true
      etcd: true
      configReloaders: true
      general: true
      k8s: true
```

and then do:

```yaml
  valuesInline:
    {%- if DRYDOCK_PROMETHEUS_VALUES %}
    {{ DRYDOCK_PROMETHEUS_VALUES | indent(4) }}
    {%- endif %}
```

Reply:
Good suggestion! I've made some changes following them.